<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_Forward">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>105</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Forward</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Backward">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>105</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Backward</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_State">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>73</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Right">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>105</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Left">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>105</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
